libs/designer-ui/src/lib/unitTesting/outputMocks/outputMocks.tsx

```diff
@@ -59,7 +48,36 @@ export interface OutputsField extends SettingProps {
   onValueChange?: ChangeHandler;
 }
 
-export const OutputMocks = ({ isMockSupported, nodeId, onActionResultUpdate, outputs, mocks }: OutputMocksProps) => {
+export interface OutputMocksProps {
+  isMockSupported: boolean;
+  nodeId: string;
+  onActionResultUpdate: ActionResultUpdateHandler;
+  outputs: OutputsField[];
+  mocks: OutputMock;
+  errorMessage: string;
+  onErrorMessageChange: (value: string) => void;
+  errorCode: string;
+  onErrorCodeChange: (value: string) => void;
+  onMockUpdate: any;
```

Comment:
NIT: Is there any chance you can set the right type, I think you got it right in the `errorDetails.tsx`